x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -2235,6 +2235,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.events.security.provider_etw',
+    first_supported_version: '8.19.0',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.events.security.provider_etw',
+      {
+        defaultMessage:
+          'Controls whether Microsoft-Windows-Security-Auditing ETW provider is enabled for security events collection. Set to false to disable the provider. Default: true.',
```

Comment:
> Set to false to disable the provider

If set to false, Endpoint will (for now) fall back to the event log provider, right?  If so, can we state that?  I can imagine users setting this to false then being confused as to why they're still receiving security events.